### PR TITLE
Expose counter for getnativecontracts

### PR DESCRIPTION
**Pull request by roman-khimov**
https://github.com/neo-project/neo/issues/2905 is incomplete without it.

### CODE REVIEW

**Review by vncoelho — COMMENTED** [review]
Looks good to me

**Review by vncoelho — COMMENTED** [review]
I will compile everything and test here today or tomorrow.

File: src/RpcClient/Models/RpcNativeContract.cs [hunk 1]
```diff
@@ -1,47 +0,0 @@
-// Copyright (C) 2015-2024 The Neo Project.
```
Comment (cschuchardt88) [inline src/RpcClient/Models/RpcNativeContract.cs:1]: Please put this back.

**Review by cschuchardt88 — CHANGES_REQUESTED** [review]
This breaks `RpcClient`, please update `RpcClient`.

**Review by shargon — APPROVED** [review]
Simpler and better

**Review by Jim8y — APPROVED** [review]

**Review by vncoelho — APPROVED** [review]

**Review by AnnaShaleva — COMMENTED** [review]
I'm too slow, but it's a really nice way.

**Pull request merged**